src/api-errors.js

```diff
@@ -77,3 +77,44 @@ export const invalidParameters = create(10, (message, errors) => {
     message: message || 'invalid parameters'
   }
 })
+
+export const vmMissingPvDrivers = create(11, () => ({
+  message: 'missing PV drivers'
+}))
+
+export const vmIsTemplate = create(12, () => ({
+  message: 'VM is a default template'
+}))
+
+export const vmBadPowerState = create(13, params => ({
```

Comment:
We're gonna use this specific errors for now, but I'd like you to note (in comments) the ideas for more generic errors we have, like:

```
incorrectState {
  objectId: 'af43e227-3deb-4822-a79b-968825de72eb',
  property: 'power_state',
  actual: 'Running',
  expected: 'Halted'
}
```